websocket_js.go

```diff
@@ -0,0 +1,217 @@
+package websocket // import "nhooyr.io/websocket"
```

Comment:
Shouldn't this file have the build tag: `+build js,wasm`?

Reply:
Files with _js are automatically compiled only for GOOS=js.

Reply:
👌 Combined with the other thread about not needing `wasm` in the build tags, this should be fine.